data/json/vehicle_groups.json

```diff
@@ -75,6 +75,19 @@
       [ "wood_rowboat_double", 1500 ]
     ]
   },
+  {
+    "type": "vehicle_group",
+    "id": "lake",
```

Comment:
This group should probably only have the DUKW and your Cabin Cruiser in it.  They are pretty close in size and far larger then any other boats.  Then, find a good spot for large boats to anchor.  Your 0,0 point placement for the cruiser may make this incompatible with other boats for a vehicle group though.

Reply:
I've changed the 0,0 point for the boat, this fixes the error and will make spawning a bit easier. The controls is now the 0, 0 point.

Reply:
That's now been updated and should be committed now, all that is in the group or will be added to the group now are lake vehicles such as my cabin cruiser, the DUKW, and shortly a sailboat :)